### DISCUSSION TITLE
Playing all games on accounts

### DISCUSSION THREAD

**Discussion by bolexator** [chr233/ASFEnhance#301] [功能建议 / Ideas]
Is it possible to make a function that would launch one by one the games that are on the account?
For example, there are 5 accounts, all of them have different games and different amounts of them. On the command **!play all** would run games that are on a particular account in turn and would play for 2 hours (can be prescribed in the settings). One account would play its 10 games in a circle, the other its 15 games, and so on.

Можно ли сделать функцию, которая бы запускала по очереди игры, которые есть на аккаунте?
Например есть 5 аккаунтов, у всех разные игры и разное их количество. По команде **!play all** запускались бы игры которые есть на конкретном аккаунте по очереди и игрались бы в течении 2 часов (можно прописать в настройках). Один аккаунт играл бы свои 10 игр по кругу, другой свои 15 игр, и так далее.

**Comment by woctezuma** [comment]
For info, you can play [up to 32 games](https://github.com/JustArchiNET/ArchiSteamFarm/wiki/FAQ#can-asf-farm-multiple-games-at-once) at the same time.

**Reply by bolexator** [reply]
> For info, you can play [up to 32 games](https://github.com/JustArchiNET/ArchiSteamFarm/wiki/FAQ#can-asf-farm-multiple-games-at-once) at the same time.

Right. And that's fine, even if it's 10 games at a time. My proposal is not for farming, but rather to gain hours (you can look at it this way). There are accounts that have more than a thousand games in their library, and other accounts also have thousands of games, but others. Run them in manual mode using universal for all accounts **!play** will not work, in view of the large differences in the library. Each account has its own list of appIDs.
The idea is that on a certain command, accounts will play each its own library of games. And so without stopping until you restart ASF or the usual command **!resume**.

**Comment by josie564** [comment]
It sounds like you're looking to automate the process of running games across multiple accounts with varying libraries in a way that each account runs its own set of games continuously. This approach would indeed streamline managing large libraries and maintain activity for hours.

While running multiple games manually or with a universal command might not work due to the differences in appIDs, implementing a system where each account can independently play through its unique library could be a practical solution. This would ensure that each account is engaged with its own games, maximizing usage and efficiency without requiring constant manual intervention.

For anyone interested in exploring more about game management and automation, you might find additional resources and tools useful at <a href="https://jojoy-ios.com/">Jojoy Website</a> . This website is dedicated to providing insights and solutions related to gaming, which could complement your approach to automating game play across multiple accounts.

**Reply by woctezuma** [reply]
This reads like AI-generated spam.
It does not help that your GitHub account was created 45 minutes ago, i.e. 8 minutes before your post.

**Reply by bolexator** [reply]
> This reads like AI-generated spam. It does not help that your GitHub account was created 45 minutes ago, i.e. 8 minutes before your post.

Probably an advertisement for a certain resource. But the thought at the beginning of the post is correct. This is exactly the kind of functionality that is missing.
